app/models/ability.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+class Ability
+  include CanCan::Ability
+
+  def initialize(user)
+    # Define abilities for the passed in user here. For example:
+    #
+      user ||= User.new # guest user (not logged in)
+      if user.is_admin?
+        can :manage, :all
+      end
+    #
+    # The first argument to `can` is the action you are giving the user
```

Comment:
Is a good habit to get rid of these instructional comments in your projects. This will help clear up you code and is good practice. Especially on projects you'd like to showcase to employers.